src/js/angular/core/services/license.service.js

```diff
@@ -83,6 +75,14 @@ function licenseService($window, $document, LicenseRestService, $translate) {
         return _license;
     };
 
+    const licenseErrorMsg = () => {
+        if (!_license?.present && _license?.message === "No license was set") {
+            return $translate.instant('no.license.set.msg');
+        } else {
+            return $translate.instant("error.license", {message: _license?.message});
```

Comment:
do this actually translate? Does `instant` interpolate  `{{'common.error'|translate}}`?

Reply:
Yes